securedrop_client/logic.py

```diff
@@ -413,6 +415,15 @@ def __init__(  # type: ignore [no-untyped-def]
         ):
             os.chmod(self.last_sync_filepath, 0o600)
 
+    def on_main_queue_updated(self, num_items: int) -> None:
+        if num_items > 0:
+            status_text = ngettext(
+                "Retrieving 1 message", "Retrieving {message_count} messages", num_items
```

Comment:
I don't know what a .pot file is but could this bit make use of those `msgid_plural` and `msgid` fields?

Reply:
This is a gettext "portable object template" file. There's a [pre-commit hook](https://github.com/guardian/securedrop-client/blob/36d426179f99c68b9f679113042a135663d0e37a/.githooks/pre-commit#L4) which extracts strings that need to be localized (those passed to `gettext` or `ngettext`) from .py files and adds them to the .pot file.
A translator would then come along and add translations, which are stored in a separate .po file for each target language. More info here: https://babel.pocoo.org/en/latest/messages.html